### ISSUE TITLE
[Bug]: com.intellij.serviceContainer.AlreadyDisposedException

### ISSUE THREAD

**Issue by brendandebeasi** [laravel-idea/plugin#435]
### Bug description

Getting this error upon starting PHPStorm
```com.intellij.serviceContainer.AlreadyDisposedException: Cannot create com.intellij.openapi.project.DumbService because container is already disposed: Project(name=studioflow, containerState=DISPOSE_COMPLETED, componentStore=/Users/brendan/git/studioflow) (disposed)
	at com.intellij.serviceContainer.ComponentManagerImpl.throwContainerIsAlreadyDisposed(ComponentManagerImpl.kt:522)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:498)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:471)
	at com.intellij.openapi.project.DumbService.getInstance(DumbService.java:190)
	at com.intellij.openapi.project.DumbService.isDumb(DumbService.java:61)
	at com.laravel_idea.plugin.helperCodeGeneration.g.d(g.java:105)
	at com.laravel_idea.plugin.helperCodeGeneration.g.a(g.java:17)
	at com.laravel_idea.plugin.helperCodeGeneration.g$b.run(g$b.java:41)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)```



### Plugin version

4.4.2.211

### Operating system

MacOS

### Steps to reproduce

Unclear how this occurs, but it happens more than once

### Relevant log output

_No response_

**Comment by adelf** [comment]
Hi, Brendan. I did some fixes around helper code generation in 4.4.3 version. It might help.

**Comment by brendandebeasi** [comment]
Shoot, didn't realize I wasn't running the latest. Will close and reopen if I can reproduce on latest.
